filter/etc/pam.d/rspamd

```diff
@@ -0,0 +1,5 @@
+#%PAM-1.0
+auth       required     pam_nologin.so
+auth       include      password-auth
```

Comment:
The example in succed_if module:

    auth required pam_succeed_if.so quiet user ingroup wheel